tests/phpunit/integration/Modules/Analytics_4Test.php

```diff
@@ -2609,13 +2653,13 @@ protected function create_fake_http_handler( $property_id ) {
 					);
 
 				case "/v1alpha/properties/$property_id/audiences":
-					$audience = new GoogleAnalyticsAdminV1alphaAudience();
-					$audience->setName( "properties/$property_id/audiences/1" );
-					$audience->setDisplayName( 'Test' );
-					$audience->setDescription( 'Description' );
+					$fixture = json_decode(
```

Comment:
We might want to extend our `TestCase` base class with a `get_fixture()` method, or suchlike at some point but this seems fine for now.

Reply:
I agree, I will open an issue to address this. Thank you Tom!

Reply:
Why not use the way it was implemented before? I can see why we might want to use a fixture due to the verbose definition of filterClauses or something else but we should address that with a helper method which allows for generating the code we want with some inputs to keep the use intuitive to understand in the test. Fixtures hide details that are often used for assertions and are easy to create in a way which is not representative of the real thing.

Reply:
Well, personally I feel it's cleaner this way. The intention of the handler is to model returning the JSON response from the API, so it seems appropriate to use a JSON fixture as we do for similar cases in the JS test suite. The fixture file in this case _is_ representative of the real thing, containing examples of the various audience types that are derived from a real response, and it's also used in the JS tests, so it seems reasonable to me to use it across both suites. We _could_ introduce a helper but I don't think it would add much in this case, having essentially two different approaches to how we manage our mock audience response data.

Not a major issue in my view, but that's my take on it.